### ISSUE TITLE
Implement structured data search types for SEO optimization

### ISSUE THREAD

**Issue by iamareebjamal** [fossasia/open-event-frontend#6106]
https://developers.google.com/search/docs/data-types/event

https://developers.google.com/search/docs/guides/generate-structured-data-with-javascript

**Comment by mariobehling** [comment]
Thanks! Very important.

**Comment by himanshujaidka** [comment]
I am working

**Comment by Creatoon** [comment]
Working on this.

**Comment by takshch** [comment]
I am working on this.

**Comment by takshch** [comment]
Testing are remaining. I am working on it.

**Comment by takshch** [comment]
This issue requires ember-mirage for mocking the server for the testing purpose. 
Ember-mirage needs to be configured for the every existing tests. Ember-mirage configuration for existing tests is separate issue or feature which is need to done first for adding tests for this issue of structured data.
So, PR of this issue does not have tests as this issue has dependency on the feature or issue of 'configuration of ember-mirage'. 
So, PR of this issue is allowed to go for review or not?
@mariobehling @iamareebjamal

**Comment by takshch** [comment]
My exams are completed. So, i again started working on this issue.

**Comment by takshch** [comment]
I stopped working on this issue because in few weeks, FossAsia is going to change its tech stack. 
Thanks.
